tests/groupScoreAndSync.test.js

```diff
@@ -27,6 +27,16 @@ const questions = _.keyBy(question, 'qid');
 describe('assessment instance group synchronization test', function () {
   this.timeout(10000);
 
+  const storedConfig = {};
+  before('store authenticated user', () => {
+    storedConfig.authUid = config.authUid;
+    storedConfig.authName = config.authName;
+    storedConfig.authUin = config.authUin;
+  });
+  after('unset authenticated user', () => {
+    Object.assign(config, storedConfig);
+  });
```

Comment:
This whole test had a side effect of mutating `storedCofig.auth*`. We now properly restore the original values like the other tests.

Reply:
Nice cleanup!